source/triggers/authentication-triggers.txt

```diff
@@ -103,9 +102,9 @@ Authentication Triggers have the following configuration options:
    * - :guilabel:`Linked Function`
 
      - The name of the {+service-short+} Function that the trigger
-       executes whenever it fires. The trigger passes the :ref:`authentication
-       event object <authentication-events>` that caused it to fire as the only
-       argument to this function.
+       executes whenever it fires. An :ref:`authentication
```

Comment:
Change line 95 "Trigger Type" to:

> For authentication triggers, set this value to ``AUTHENTICATION``.

Reply:
Change line 104 "Linked Function":

Remove {+service-short+} and lowercase Function. Change "whenever" to "when".

Reply:
Change line 118 "Providers":

Remove "will":
> The trigger only listens for...